src/erlang_ls_completion_provider.erl

```diff
@@ -39,13 +39,14 @@ handle_request({completion, Params}, State) ->
    , <<"textDocument">> := #{<<"uri">> := Uri
                             }
    } = Params,
-  {ok, Document} = erlang_ls_db:find(documents, Uri),
+  {ok, #{text := Text}} = erlang_ls_db:find(documents, Uri),
```

Comment:
I think this is breaking the opaqueness of the `document()` type. Why not use the `erlang_ls_document:text/1` function?

Reply:
Nice catch, will fix.